src/app/Recordings/ActiveRecordingsTable.tsx

```diff
@@ -98,8 +99,9 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
   const handleRecordings = React.useCallback((recordings) => {
     setRecordings(recordings);
     setIsLoading(false);
+    recordings.length === 0 ? setIsEmpty(true) : setIsEmpty(false);
```

Comment:
Just a style thing so feel free to take it or leave it, but this could be written as `setIsEmpty(recordings.length === 0)` or even `setIsEmpty(!recordings.length)`

Reply:
Or just a standard if/else, rather than this ternary where the expression value isn't actually used